markdown/preface.markdown

```diff
@@ -159,20 +160,24 @@ Although this book will focus on NumPy and SciPy,
 it is the many surrounding packages that make Python a powerhouse for
 scientific computing.
 
-
 ## The Great Cataclysm: Python 2 vs. Python 3
 
 In your Python travels, you may have already heard a few rumblings about which version of Python is better.
 You may have wondered why it's not just the latest version.
 
-At the end of 2008, the Python core developers released Python 3.0, a major update to Python with better Unicode (international) text handling, type consistency, and streaming data handling, among other improvements.
-To quote Douglas Adams in The Hitchhiker's Guide to the Galaxy, "this has made
-a lot of people very angry and been widely regarded as a bad move."
```

Comment:
I think this is unnecessarily picky. What's wrong with using at quote here? I think it works nicely, particularly with your additions below.